client/src/components/Header.js

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'styled-components';
+
+const Heading = styled.h1`
```

Comment:
The header text has these properties on the Figma. Can you make them match closer to the reference design? 
![image](https://user-images.githubusercontent.com/47200969/124698681-2f9dc580-de9e-11eb-919e-fd91ee0ba4f0.png)